opcua/client/client.py

```diff
@@ -86,7 +86,7 @@ class Client(object):
     the raw OPC-UA services interface.
     """
 
-    def __init__(self, url, timeout=4):
+    def __init__(self, url, timeout=4, secure_channel_timeout=3600000, session_timeout=3600000):
```

Comment:
it is not because the parameters are not passed to constructor that you cannot set them, they are not hidden. They were simply not taken as arg to constructor becasue they are not so common. but any more serious application should change a bunch of them like product_uri, certificates, etc....

Reply:
otherwise all changes are fine

Reply:
Should I leave out the parameters and add the documentation elsewhere?

Reply:
I tend to think we should not have these parameters as arguments. They are not more important than the others

Reply:
Where should I document them then?

Reply:
Inside constructor string is fine. Just write that they need to be set up after constructing the object